dashboard/prisma/schema.prisma

```diff
@@ -13,17 +13,21 @@ datasource db {
 model Account {
   id                 String    @id @default(cuid())
   userId             String
-  providerType       String
-  providerId         String
+  type               String
+  provider           String
   providerAccountId  String
-  refreshToken       String?
-  accessToken        String?
-  accessTokenExpires DateTime?
+  refresh_token      String?
+  access_token       String?
+  expires_at         Int?
+  token_type         String?
+  scope              String?
```

Comment:
Should these be prefixed with `token` to indicate they're related to the `access_token` and not the whole `Account` record?